Use new key binding system in cgame.

slipher opened the pull request:
See https://github.com/DaemonEngine/Daemon/pull/75 for the engine side.

TODO: Fix license headers of added file.

slipher commented on src/cgame/rocket/rocketKeyBinder.h:
```diff
@@ -188,7 +183,7 @@ class RocketKeyBinder : public Rocket::Core::Element, public Rocket::Core::Event
 			Rocket::Core::String label;
 		} labels[] = {
-			{ 0, "spectators" },
+			{ 3, "spectators" },
```
I added spectator binds in one of the earlier PRs.
https://github.com/DaemonEngine/Daemon/pull/62/commits/e4bf20cd0c126fd5bde14e87c1e49147dc18db0f

slipher commented on src/cgame/cg_tutorial.cpp:
```diff
@@ -25,34 +25,36 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
-} bind_t;
+	std::vector<Keyboard::Key> keys;
+};
+}
```
Will fix.

slipher commented on src/cgame/cg_key_name.cpp:
```diff
@@ -0,0 +1,56 @@
+
+using Keyboard::Key;
+
+std::string KeyDisplayName(Key key) {
```
Uh, I don't know.

DolceTriade replied:
I'd prefer to at least to keep CG_Key or something so that the code looks visually similar as the other code. One day we can refactor all the CG_ stuff into a namespace.

slipher commented:
All right, I have changed the license to GPL on new files and addressed comments. Now only to wait for https://github.com/DaemonEngine/Daemon/pull/75 to be reviewed.
